src/GeneratedHydrator/Configuration.php

```diff
@@ -220,4 +225,20 @@ public function getClassNameInflector()
 
         return $this->classNameInflector;
     }
+
+    /**
+     * @param \PhpParser\NodeVisitor[] $visitors
+     */
+    public function setCustomVisitors(array $customVisitors)
```

Comment:
I'm not sure but Maybe can be a good ideia pass $customVisitors by constructors.